lib/ansible/modules/cloud/openstack/os_stack.py

```diff
@@ -20,7 +20,9 @@
 short_description: Add/Remove Heat Stack
 extends_documentation_fragment: openstack
 version_added: "2.2"
-author: "Mathieu Bultel (matbu), Steve Baker (steveb)"
+author:
+  - "Mathieu Bultel (@matbu)"
```

Comment:
If we're changing this now, maybe we should remove the useless quotes as well ?
Don't mind editing these myself, if you wouldn't bother.

Reply:
Can discuss this outside of this PR.

Reply:
Sure.